metadata-ingestion/src/datahub/ingestion/transformer/extract_ownership_from_tags.py

```diff
@@ -111,5 +156,7 @@ def transform_aspect(
                 ),
             )
         )
-
-        return None
+        if not self.config.replace_existing:
```

Comment:
this doesn't make sense - we should always `return aspect`

Reply:
Even I thought so. But previous behavior was removing tag metadata from ingestion after extracting ownership from it.